src/test/ui/binop/binop-mul-i32-f32.rs

```diff
@@ -0,0 +1,5 @@
+fn foo(x: i32, y: f32) -> f32 {
+    x * y //~ ERROR cannot multiply `i32` by `f32`
+}
+
+fn main() {}
```

Comment:
Do you need this new test case? This is tested in plenty of other places.

Reply:
I couldn't find a place where a simple case of the error was tested, and I wanted a sanity check since this is behaving weirdly. But if you think I should remove it once we figure out what's going awry, I guess I could. I just wanted extra coverage while I figured it out :)

Reply:
I think I'm going to keep this test unless there are significant objections since this helped catch a spot that I didn't know to update.